curvefs/src/client/s3/disk_cache_manager.cpp

```diff
@@ -32,6 +32,26 @@ namespace curvefs {
 
 namespace client {
 
+/**
+ * use curl -L mdsIp:port/flags/avgFlushBytes?setvalue=true
+ * for dynamic parameter configuration
+ */
+static bool pass_uint64(const char*, uint64_t) { return true;}
+DEFINE_uint64(avgFlushBytes, 83886080, "the write throttle bps of disk cache");
+DEFINE_validator(avgFlushBytes, &pass_uint64);
+DEFINE_uint64(burstFlushBytes, 104857600, "the write burst bps of disk cache");
+DEFINE_validator(burstFlushBytes, &pass_uint64);
+DEFINE_uint64(burstSecs, 180, "the times that write burst bps can continue");
+DEFINE_validator(burstSecs, &pass_uint64);
+DEFINE_uint64(avgFlushIops, 100000000, "the write throttle iops of disk cache");
+DEFINE_validator(avgFlushIops, &pass_uint64);
+DEFINE_uint64(avgReadFileBytes, 83886080,
+  "the read throttle bps of disk cache");
+DEFINE_validator(avgReadFileBytes, &pass_uint64);
+DEFINE_uint64(avgReadFileIops, 100000000,
+  "the read throttle iops of disk cache");
+DEFINE_validator(avgReadFileIops, &pass_uint64);
```

Comment:
if you want to modify throttle parameters at runtime, only defining these flags is not enough. you should also implement an event mechanism or a watcher, so that, when flags is been modified, you can update throttle parameters through new limits.

Reply:
has already done